### ISSUE TITLE
Specify confidence in the results

### ISSUE THREAD

**Issue by cpaasch** [network-quality/draft-ietf-ippm-responsiveness#55]
Specify how we can determine the confidence in the results.

**Comment by hawkinsw** [comment]
Just to record the notes that we made during our last meeting:

1. Specify that there are three confidence intervals
2. Specify that a result will be assigned a confidence interval based on how its measurement was made (not necessarily on the score or any variability in results from individual probes)
3. Broken down like
- High: LGCs (via HTTP2 (with SSL)  were used to saturate the link; probes collected data (via HTTP2 (with SSL) ) on an LGC connection and in the background (by establishing new HTTP2 (with SSL) connections from scratch each time)
- Medium: HTTP (without SSL) was used to saturate the link; probes collected data in the background (by establishing new HTTP (_with_ SSL) connections from scratch each time)
- Low: "All HTTP (without SSL) all the time"

That is at least what I *thought* I recalled from the last meeting. @cpaasch is this right?
